config/onprem-iso-fcc.yaml

```diff
@@ -3,8 +3,7 @@ version: 1.1.0
 passwd:
   users:
     - name: core
-      ssh_authorized_keys:
-        - "replace-with-your-ssh-public-key"
+      ssh_authorized_keys: ["replace-with-your-ssh-public-key"]
```

Comment:
Any reason why you canged this? Asking because old and new are both an array.

Reply:
With the old syntax the generated yaml would look like:
```
ssh_authorized_keys:
        - "key1", "key2"
```
which is not correct.

With the new syntax the generated yaml will look like:
```
ssh_authorized_keys: ["key1", "key2"]
```

which is correct yaml.

Reply:
@machacekondra will this break current onprem deployments?

Reply:
I don't see a use case, where it could break the current on prem deployments.

Reply:
+1

Reply:
For what is worth, I would have used the following as the example (so just adding a item to the array)

```
ssh_authorized_keys:
    - "key1"
    - "key2"
```

Reply:
iirc, YAML should support the two syntax
```
ssh_authorized_keys: ["item1", "item2"]

or

ssh_authomized_keys:
 - item1
 - item2
```

second one should be better for long outputs

Reply:
I agree, but I've changed for simpler code generation.